lib/aruba/platform.rb

```diff
@@ -230,6 +228,16 @@ def which(program, path = ENV['PATH'])
     # rubocop:enable Metrics/MethodLength
     # rubocop:enable Metrics/CyclomaticComplexity
 
+    # return true if the OS running is a Unix system according to FFI
+    def unix?
+      FFI::Platform.unix?
+    end
```

Comment:
I can see taking this out, since the classes are set up such that the ```Basic*``` class has the default behavior and that behavior is for unix systems.  Thus we only care about *not unix* systems, which == *windows*.  (And so the ```Windows*``` child classes have the variant behavior).    The value of leaving it in is so that the intentions are explicit to anyone reading the code.  It's clear not only that we care about Windows (because we have to do something different), but also that the default behavior is for *unix* systems.  In this case, since the class names are ```Unix*``` I'd say that the intention is already clear enough.
(But if it's left in, or just for future reference:  ```FFI::Platform.unix?``` does return true for all of those (```mac?``` ```solaris?``` etc.). It actually just returns ```!windows?``` -- which is exactly our case.)